run_all_script.R

```diff
@@ -0,0 +1,101 @@
+#' # Test RMSE versus Training Temperature Profiles
+#' 
+#' This script fetches, processes, and plots data to compare Test RMSE versus Training 
+#' temperature profiles. 
+#' 
+#' 
+#' ## Header
+#' 
+#' Load libraries
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+library(devtools)
+
+#' Clear environment (optional)
+#' 
+rm(list = ls())
```

Comment:
These lines can come out. Jenny Bryan from rstudio has a helpful write up on this [here](https://www.tidyverse.org/blog/2017/12/workflow-vs-script/)